src/components/sale/SaleCollectionAdminCard.tsx

```diff
@@ -347,6 +347,22 @@ const SaleCollectionAdminCard: React.VFC<{
               </div>
             </div>
             <Divider />
+            {v.options?.giftPlans?.length > 0 &&
+              v.options?.giftPlans.map((v: any) => {
+                return (
+                  <>
+                    <div className="row">
+                      <div className="col-2">
+                        <ProductTypeLabel productType="GiftPlan" />
```

Comment:
adding a new component `<TokenTypeLabel type="GiftPlan" />` will be better

Reply:
had TokenTypeLabel